src/foam/core/pool/scripts.jrl

```diff
@@ -0,0 +1,19 @@
+p({
```

Comment:
and by use COREService stop, start, reload, you don't need this explicit script. 
I would expect there is alread a COREService stop, reload script.